Headers/DebugServer2/Host/Linux/ExtraWrappers.h

```diff
@@ -86,21 +86,26 @@ struct xfpregs_struct {
 static inline int posix_openpt(int flags) { return ::open("/dev/ptmx", flags); }
 #endif // !HAVE_POSIX_OPENPT
 
-#if !defined(PLATFORM_ANDROID)
+// Do not use PLATFORM_ANDROID here. We want to test if we're using the Android
+// toolchain, not if we're building for the Android target. In some cases, we
+// build Tizen binaries with the Android toolchain.
+#if !defined(HAVE_GETTID)
 // Android headers do have a wrapper for `gettid`, unlike glibc.
 static inline pid_t gettid() { return ::syscall(SYS_gettid); }
-#endif // !PLATFORM_ANDROID
+#endif // !HAVE_POSIX_OPENPT
```

Comment:
Small thing: This marks the end of !defined(HAVE_GETTID), not !defined(HAVE_POSIX_OPENPT)